### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by aakbarkhan**
## Description:

- Set as re-usable components.
- Add React Router and set two Routes and Links for the app's navigation:
- Books - the default view
- Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)
- Should have a form for adding a book (no functionality yet)
- Categories- Should display "Under construction" text only.

Styling is not required at this point.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const categories = () => (
+  <div>
+    <p>Under construction</p>
+  </div>
+);
+
+export default categories;
```
Comment (TedLivist) [inline src/redux/categories/categories.js]: - [ ] You created your Category component under the `redux` directory. All components should live under the `components` directory. This `redux` directory is solely meant for redux configurations. Kindly extract this component on to the **components** folder

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import AddBooks from '../../components/addBooks';
+
+const Books = () => (
+  <div>
+    <Book />
+    <AddBooks />
+  </div>
+);
+
+export default Books;
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [ ] Kindly extract this **Books** component into the components folder.

File: src/components/book.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Button from './button';
+
+const Book = () => (
```
Comment (TedLivist) [inline src/components/book.js]: - [ ] The **Book** component should take in a book prop data and render the book title and author

File: README.md [hunk 4]
```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (TedLivist) [inline README.md:2]: - [ ] You have correctly used the recommended README template. But it is still a dummy file. Kindly update your README with the necessary information regarding this project.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @aakbarkhan,

Good job so far!
✔️ Neat code
✔️ Good implementation of SOLID principle
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 2 above)
Comment (aakbarkhan) [reply src/redux/books/books.js]: image

Print screen from the task description.
Could you please provide more info?

File: src/redux/categories/categories.js (hunk 1 above)
Comment (aakbarkhan) [reply src/redux/categories/categories.js]: image

Print screen from the task description.
Could you please provide more info?

Comment (TedLivist) [reply src/redux/categories/categories.js]: Hey @aakbarkhan as you can see there are two folders (**components** and **redux**). The components folder should contain all your components. And the redux folder should contain redux-related configurations (like actions, action types, reducers, and so on). What you have in **`redux/categories/categories`** is a component, and so it should be under the components folder.

Comment (aakbarkhan) [reply src/redux/categories/categories.js]: Thank you.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi @aakbarkhan     :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**